readthedocs.yml

```diff
@@ -1,7 +1,19 @@
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: doc/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: all
+
 python:
-    version: 3.5
-    setup_py_install: true
-    pip_install: true
-    extra_requirements:
-        - tests
-        - docs
+    version: 3.7
```

Comment:
@weikang9009 here @renanxcortes is using Python 3.7, which [`giddy`](https://github.com/pysal/giddy/blob/master/readthedocs.yml) also uses. Should [`spaghetti`](https://github.com/pysal/spaghetti/blob/master/readthedocs.yml) also be using 3.7? Currently, it uses 3.6.

Reply:
I do not think it matters much between using 3.7 and 3.6 since the package supports both